Short-Answer/Algorithms_Answers.md

```diff
@@ -2,14 +2,32 @@
 
 ## Exercise I
 
-a)
+a) O(n) because as the input increases, the run time will increase at the same rate
 
 
-b)
+b) O(n log n) because the for loop is O(n): as the input increases, the run time grows at the same rate.
+  The while loop inside the for loop is O(log n) because the run time grows at a slightly slower rate as the input increases. Therefore, O(n log n).
```

Comment:
This is a better example of a more thorough analysis of why the algorithm is `O(n log n)`. You're telling me what happens to n inside both loops.

Overall, really nice job with these!